private-templates-service/client/src/components/Common/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -47,29 +47,26 @@ export function renderAdaptiveCard(template: Template): any {
   try {
     // Render the card to an HTML element
     let renderedCard = adaptiveCard.render();
+
     return renderedCard;
   } catch (e) {
     return <div>Error</div>;
   }
 }
 
-function setContextRoot(data: string, context: ACData.EvaluationContext) {
+function setContextRoot(data: object, context: ACData.EvaluationContext) {
   try {
-    let dataString = JSON.stringify(data);
-    let dataJSON: JSON = JSON.parse(dataString);
-    context.$root = dataJSON;
+    context.$root = data;
   } catch (e) {
-    console.log(e);
   }
```

Comment:
Since it is just an assignment now, I guess there is no need function as you are also not doing anything with try catch.

Reply:
Removed!